BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppCoordinator.swift

```diff
@@ -160,7 +161,37 @@ final class AppCoordinator: Coordinator {
                         for: [GiniCapturePage(document: document, error: nil)])
                 } catch  {
                     self.rootViewController.showErrorDialog(for: error, positiveAction: nil)
-                }
+                }            // When a document is imported with "Open with", a dialog allowing to choose between both APIs
```

Comment:
[Suggestion] maybe we can remove this comment that is already added in top of this `if` statement. Also, same suggestion related with self.